ci/concourse/terraform-dockerimage-pipeline.yml

```diff
@@ -0,0 +1,191 @@
+resources:
+- name: terraform-ibm-ha-repo
+  type: git
+  icon: github
+  source:
+    uri: https://github.com/fshuva/ibm-fortigate-AP-HA-terraform-deploy.git
+    branch: concourse_setup
+- name: terraform-0-13-7
+  type: docker-image
```

Comment:
should prefer the newer resource type 'registry-image'
See https://github.com/concourse/registry-image-resource#comparison-to-docker-image-resource

Reply:
Done